lib/LoginError.js

```diff
@@ -40,6 +40,7 @@ function LoginError(status, details) {
   err.status = status;
   err.name = obj.code;
   err.code = obj.code;
+  err.require_recaptcha = obj.require_recaptcha;
```

Comment:
Thanks for the contribution. Can this be done using the `code` attribute? Thanks!